src/components/App/App.js

```diff
@@ -9,28 +9,51 @@ class App extends React.Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      selectedId: 0
+      movies: [],
+      selectedDetails: null,
+      error: false,
     };
   }
 
+  componentDidMount() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => {if (response.ok) {return response.json()}})
+      .then(data => this.setState({movies: data.movies}))
+      .catch(error => this.setState({error: true}))
+  }
+
   selectMovie = id => {
-    this.setState({selectedId: id})
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+      .then(response => {if (response.ok) {return response.json()}})
+      .then(data => this.setState({selectedDetails: data.movie}))
+      .catch(error => this.setState({error: true}))
+  }
+
+  clearSelection = () => {
+    this.setState({selectedDetails: null})
+  }
+
+  chooseContent() {
```

Comment:
Instead of putting this into a separate function with if/else blocks, could you refactor the render's return and use conditional formatting syntax?